rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -620,6 +620,14 @@ final class Binary implements J, Expression, TypedTree {
 
         JLeftPadded<Type> operator;
 
+        public JLeftPadded<Type> getLeftPaddedType() {
+            return operator;
+        }
+
+        public Binary withOperator(JLeftPadded<Type> leftPadded) {
+            return getPadding().withOperator(leftPadded);
+        }
+
```

Comment:
These methods are redundant. `J.Binary.getPadding().getType()` and `J.Binary.getPadding.getOperator()` already exist and serve these purposes, they do not need to be duplicated.

Reply:
resolved